styles/_bar.styl

```diff
@@ -0,0 +1,47 @@
+.bar
+  position absolute
+  bottom 1px
+  width 3px
+  height @width
```

Comment:
@wesbos It's totally fine to use such syntax when you have two logical bound values. In this example heigh will have a value of `3px`.

More info you can read in official doc: http://stylus-lang.com/docs/variables.html#property-lookup